src/pages/workspace/workflows/approvals/ApprovalWorkflowEditor.tsx

```diff
@@ -127,7 +127,7 @@ function ApprovalWorkflowEditor({approvalWorkflow, removeApprovalWorkflow, polic
                     interactive={!approvalWorkflow.isDefault}
                 />
 
-                {approvalWorkflow.approvers.map((approver, approverIndex) => {
+                {(approvalWorkflow.approvers.length > 0 ? approvalWorkflow.approvers : [undefined]).map((approver, approverIndex) => {
```

Comment:
I did this a bit differently than what I wrote in the proposal. What is lacking from the proposal is that the Approver won't have a field required error text and it will still show in the "Additional approver" field.

So, I just put an undefined item here, the index will be 0 and I update the error assignment above to assign the error to "approver-0" instead of "additionalApprover".